assets/scss/base/layouts/_layouts.scss

```diff
@@ -93,6 +93,12 @@ body {
   width: $full-width;
 }
 
+.full-width--small {
+  @include media(mobile) {
```

Comment:
Rather than a helper can this not be added to the component? If @hmrc/designers would like all buttons to go full width on mobile I would add it there rather than creating a new helper class.